src/components/Calculator.js

```diff
@@ -1,43 +1,44 @@
-              <th><button type="button" className="divide">:</button></th>
+              <th><button type="button" className="percent" onClick={type}>%</button></th>
+              <th><button type="button" className="divide" onClick={type}>÷</button></th>
             </tr>
             <tr>
-              <th><button type="button" className="one">1</button></th>
-              <th><button type="button" className="two">2</button></th>
-              <th><button type="button" className="three">3</button></th>
-              <th><button type="button" className="multiply">X</button></th>
+              <th><button type="button" className="one" onClick={type}>1</button></th>
+              <th><button type="button" className="two" onClick={type}>2</button></th>
+              <th><button type="button" className="three" onClick={type}>3</button></th>
+              <th><button type="button" className="multiply" onClick={type}>x</button></th>
             </tr>
             <tr>
-              <th><button type="button" className="four">4</button></th>
-              <th><button type="button" className="five">5</button></th>
-              <th><button type="button" className="six">6</button></th>
-              <th><button type="button" className="minus">-</button></th>
+              <th><button type="button" className="four" onClick={type}>4</button></th>
+              <th><button type="button" className="five" onClick={type}>5</button></th>
+              <th><button type="button" className="six" onClick={type}>6</button></th>
+              <th><button type="button" className="minus" onClick={type}>-</button></th>
             </tr>
             <tr>
-              <th><button type="button" className="seven">7</button></th>
-              <th><button type="button" className="eight">8</button></th>
-              <th><button type="button" className="nine">9</button></th>
-              <th><button type="button" className="plus">+</button></th>
+              <th><button type="button" className="seven" onClick={type}>7</button></th>
+              <th><button type="button" className="eight" onClick={type}>8</button></th>
+              <th><button type="button" className="nine" onClick={type}>9</button></th>
+              <th><button type="button" className="plus" onClick={type}>+</button></th>
             </tr>
             <tr>
-              <th colSpan={2}><button type="button" className="zero">0</button></th>
-              <th><button type="button" className="dot">.</button></th>
-              <th><button type="button" className="equal">=</button></th>
+              <th colSpan={2}><button type="button" className="zero" onClick={type}>0</button></th>
+              <th><button type="button" className="dot" onClick={type}>.</button></th>
+              <th><button type="button" className="equal" onClick={calculate}> =</button></th>
             </tr>
           </tbody>
         </table>
```

Comment:
- [ ] `Calculator.js` is a child component of `App.js `Props need to get passed to child components of the Calculate.js component. You need to make App.js a stateful component and then pass down the state to it's children using props . Please fix it :+1: